confidence/io.py

```diff
@@ -258,10 +266,7 @@ def loadf(
 
     def readf(fpath: Path) -> typing.Mapping[str, typing.Any]:
         try:
-            with fpath.open('r') as fp:
-                LOG.info(f'reading configuration from file {fpath}')
-                # default to empty dict, yaml.safe_load will return None for an empty document
-                return yaml.safe_load(fp.read()) or {}
+            return format.loadf(fpath)
```

Comment:
Nice

Reply:
Yeah some things got a lot simpler (though this bit just got moved, I think).